src/templates/sermons.css.map

```diff
@@ -1 +1 @@
-{"version":3,"sourceRoot":"","sources":[],"names":[],"mappings":"","file":"sermons.css"}
\ No newline at end of file
+{"version":3,"sourceRoot":"","sources":["sermons.scss"],"names":[],"mappings":"AACE;EACE;EACA;EACA;;AACA;EACE;EACA;;AAEA;EACE;EACA;EACA;EACA;EACA;;AAGF;EACE;;AAIJ;EACE;EACA;EACA","file":"sermons.css"}
```

Comment:
Do we want/need the `.css.map` committed? Is it possible to generate this in the build step and not have it in version control? I guess that applies to the `.css` too since it's coming from a `.scss` source file.

Reply:
Currently I'm using an IDE watcher to use the transcompilation, there's nothing configured in Gatsby or Netlify that would do it for me later.

Reply:
Huh, who knew?

https://www.gatsbyjs.org/docs/sass/